game/lib/game/game_runner.py

```diff
@@ -0,0 +1,110 @@
+        elif event.type == pygame.KEYDOWN and event.key == pygame.K_a:
+            self.__main_hero_horizontal_speed = -self.__hero_move_converter(self.__main_hero)
+            self.__main_hero.set_movement_left()
+        elif event.type == pygame.KEYDOWN and event.key == pygame.K_d:
+            self.__main_hero_horizontal_speed = self.__hero_move_converter(self.__main_hero)
+            self.__main_hero.set_movement_right()
+
+    def __handle_keyup_events(self, event):
+        if event.type == pygame.KEYUP and (event.key == pygame.K_w or event.key == pygame.K_s):
+            self.__main_hero_vertical_speed = IDLE_SPEED
+            self.__main_hero.reset_direction(event.key)
+        elif event.type == pygame.KEYUP and (event.key == pygame.K_a or event.key == pygame.K_d):
+            self.__main_hero_horizontal_speed = IDLE_SPEED
+            self.__main_hero.reset_direction(event.key)
+
+    def __hero_move_converter(self, hero):
+        return hero.get_move_speed() * self.__game.get_delta_time()
+
+    def __handle_number_key_event(self, event):
+        if event.type == pygame.KEYDOWN and (event.key == pygame.K_1 or event.key == pygame.K_2 or
+                                             event.key == pygame.K_3 or event.key == pygame.K_4 or
+                                             event.key == pygame.K_5):
+            value = event.key-self.__1_key_value
+            self.__main_hero.get_equipment().mark_item(value)
+
+    def __transform(self):
+        if not self.__main_hero.get_col_flag():
+            self.__map.change_bias_x(self.__main_hero_horizontal_speed)
+            self.__map.change_bias_y(self.__main_hero_vertical_speed)
+        self.__main_hero.update_position(self.__main_hero_horizontal_speed, self.__main_hero_vertical_speed)
+
+    def __draw(self):
+        self.__map.fill_screen_with_grass()
+        self.__screen.blit(self.__main_hero.get_sprite(), self.__main_hero_pos)
+        for world_object in self.__objects:
+            self.__screen.blit(world_object.get_sprite(),
+                               (world_object.get_x() - self.__main_hero.get_x()+self.__screen_size[0]/2,
+                                world_object.get_y() - self.__main_hero.get_y()+self.__screen_size[1]/2))
+            # world_object.draw_collision_rect(self.__screen, self.__main_hero.get_x()-self.__screen_size[0]/2, self.__main_hero.get_y()-self.__screen_size[1]/2,
+            #                                  self.__main_hero.get_width(), self.__main_hero.get_height(),
+            #                                  self.__main_hero.get_center_x(), self.__main_hero.get_center_y())
+
+        marked_index = self.__main_hero.get_equipment().get_marked_index()
+        for y in range(0, 5):
+            if y == marked_index:
+                self.__screen.blit(self.__marked_slot_sprite,
+                                   ((self.__screen_size[0]/2)+y*self.__eq_slot_width-self.__shift_from_middle,
```

Comment:
0 i 1 wpisać do jakiejś zmiennej typu X_INDEX, Y_INDEX i używać ich, wtedy wiadomo co to 0 i 1 znaczą. :)